public/admin/config.yml

```diff
@@ -644,3 +644,70 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+
+  - label: Locations
+    name: locations
+    label_singular: Location
+    description: description
+    format: json
+    create: true
+    files:
+      - label: States
+        name: states
+        file: src/cms-content/locations/state.json
```

Comment:
This file won't be available when we build, because we are not importing it in the app.   We will need to save it in `public` for it to be copied during the build process